From 82be015ea97e306adc3f4111974fe280175000ba Mon Sep 17 00:00:00 2001
From: Nate Drude <nate.d@variscite.com>
Date: Wed, 5 Oct 2022 15:36:25 -0500
Subject: [PATCH 3/4] iMX8M: soc: imx8mq: move TEE_LOAD_ADDR to 512mb memory
 region

The default memory location, 0xfe000000, requires 3GB of memory. Move
TEE_LOAD_ADDR to 0x56000000 to work with soms down to 512MB.

Signed-off-by: Nate Drude <nate.d@variscite.com>
---
 iMX8M/soc.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/iMX8M/soc.mak b/iMX8M/soc.mak
index 778871a..fc9ca55 100644
--- a/iMX8M/soc.mak
+++ b/iMX8M/soc.mak
@@ -70,7 +70,7 @@ else
 PLAT = imx8mq
 HDMI = yes
 SPL_LOAD_ADDR = 0x7E1000
-TEE_LOAD_ADDR = 0xfe000000
+TEE_LOAD_ADDR = 0x56000000
 ATF_LOAD_ADDR = 0x00910000
 VAL_BOARD = val
 #define the F(Q)SPI header file
-- 
2.25.1

